### PR TITLE
fix(scylla_setup): only enable swap setup when it's supported

### PR DESCRIPTION
**Pull request by amoskong**
Signed-off-by: Amos Kong <amos@scylladb.com>

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -2150,6 +2150,11 @@ def scylla_setup(self, disks):
         result = self.remoter.run('/sbin/ip -o link show |grep ether |awk -F": " \'{print $2}\'', verbose=True)
         devname = result.stdout.strip()
         extra_setup_args = self.parent_cluster.params.get('append_scylla_setup_args')
+        result = self.remoter.run('sudo /usr/lib/scylla/scylla_setup --help')
+        if '--swap-directory' in result.stdout:
+            # swap setup is supported
+            extra_setup_args += ' --swap-directory / '
+
```
Comment (aleksbykov) [inline sdcm/cluster.py:2157]: May be it is better to implement as new method?

**Comment by amoskong** [comment]
Test this PR by https://jenkins.scylladb.com/job/scylla-4.2/job/Reproducers/job/rolling-upgrade-centos7-test-clone/1/

File: sdcm/cluster.py (hunk 1 above)
Comment (fruch) [reply sdcm/cluster.py:2157]: if there were more places calling scylla_setup... , but I think there aren't

Comment (amoskong) [reply sdcm/cluster.py:2157]: > May be it is better to implement as new method?

@aleksbykov  I can send another PR to move this to a help function.

`def is_supported_option(cmd='scylla_setup', option='--swap-directory')`

**Pull request merged**